### PR TITLE
Update add -p command 

### PR DESCRIPTION
**Pull request by tanxuanyun**
### Change
- Update add -p command
- Update screenshot

### CODE REVIEW

**Review by nqt230 — APPROVED** [review]
LGTM :+1:

**Review by LiuFangrui — APPROVED** [review]
LGTM

File: docs/UserGuide.md [hunk 1]
```diff
@@ -167,24 +167,33 @@ Saves a contact person into InterNUS, from the hiring manager you liaise with
...
+    * If the internship at the specified index is already an internship for another person,
+      the linking between the person and the internship will fail with a warning, but the person will still be added.
+* `TAG` can only be alphanumeric and be one word.
+* `COMPANY` refers to the company the contact person is working at.
+* `EMAIL`,`PHONE`,`TAG`,`COMPANY` are optional as user might not have all the information of contact person.
+
+<div markdown="block" class="alert alert-info">
+:information_source: **Note:** 
+
+* Duplicate person are not allowed.
```
Comment (chengda300) [inline docs/UserGuide.md:183]: Duplicate persons

File: docs/UserGuide.md [hunk 2]
```diff
@@ -167,24 +167,33 @@ Saves a contact person into InterNUS, from the hiring manager you liaise with
...
+  A person is considered to be duplicate if there already exists a person in the list with the exact same name (case-sensitive).
```
Comment (chengda300) [inline docs/UserGuide.md:184]: to be a duplicate

File: docs/UserGuide.md [hunk 3]
```diff
@@ -167,24 +167,33 @@ Saves a contact person into InterNUS, from the hiring manager you liaise with
...
+* Adding person maintains the current sorted order of the display list (as opposed to adding to the back of the list).
```
Comment (chengda300) [inline docs/UserGuide.md:185]: Adding a person

**Review by chengda300 — COMMENTED** [review]
Some minor grammatical errors

**Pull request merged**